### DISCUSSION TITLE
Video For Begginers To Use The List

### DISCUSSION THREAD

**Discussion by ray2400** [Bon-Appetit/porn-domains#53] [Q&A] [answered]
Dear Bon Appetit, 

First of all, thank you very much for the list—it's awesome. I sincerely appreciate the work. 

Secondly, I would like to ask you to make a small video to explain how a beginner coding like me can use the list, like setting up software/systems to take advantage of this list, specifically for Mac (ideally for iPhone/iPad, too). 

I've tried it out myself -a normal internet surfer- with no success. All the user-friendly apps like Freedom, Cold Turkey, Opal etc... don't allow for a giant list of websites to be added. I've tried to use Chat GPT to understand how to use the list but it's talking about DNS, Pi-hole, so either I have a clear video or I'm stuck due to my knowledge. 

I'll be glad to offer $20 for this video; and I'm sure that many people would benefits from it. 

I understand that this is probably not the right place to post this but I don't know where else I can do this. 

Thanks for considering my offer and again, thanks for the list :) 

All the best

**Comment by CodeAlDente** [comment, marked as answer]
Hey ray2400!

First of all, I’m really glad to hear you’re finding the list useful—it’s great to know that the effort is appreciated!

Now, regarding your request, I’d recommend running Pi-Hole locally on a Raspberry Pi. It's a fantastic solution for blocking unwanted domains, including adult content, and it's surprisingly easy to set up. There are plenty of YouTube videos that walk you through the entire process, so you can get everything up and running without too much hassle.

If you’re specifically looking to integrate the list of porn domains with Pi-Hole, I’ve got a quick guide for you that can help: [Pi-Hole for Domain Blocking](https://github.com/Bon-Appetit/porn-domains/wiki/07-Pi%E2%80%90Hole-for-Domain-Blocking). This should make the process much clearer and more accessible.

I understand that diving into DNS and network configurations can be daunting, especially if you're just starting out with coding and tech setups. While I can't create a video myself, I think the combination of Pi-Hole and the guide will provide you with a solid foundation to get things going.

Thanks again for your appreciation, and I hope you find this information helpful! If you have any other questions or need further assistance, feel free to reach out. 😊

**Comment by ray2400** [comment]
Hi CodeAl Dente, 

I'm glad to see that behind a good project, there’s also a kind reply :)

I see; thanks for the reply. 

I’m planning to buy a new computer and then give it a try. However, as much as I appreciate your suggestions and am grateful for them, I think my best bet will be to hire a guy on Fiver or something like that. 

If I manage to make it, I’ll definitely let you know. 

Either way, thanks again for the project and all the best :)
